.docker/entrypoint/spec/jekyll_server_spec.rb

```diff
@@ -18,36 +18,67 @@
   end
 
   describe '#execute :serve' do
-    describe 'weird file' do
-      data_dir = File.join(__dir__, '..', '..', '..', 'tests', 'full')
-
-      before(:all) do
-        @thread = Thread.new do
-          context = Context.new('development', __dir__, data_dir)
-          jekyll_config_provider = JekyllConfigProvider.new(context)
-          context.configuration = jekyll_config_provider.provide('serve')
-          jekyll_server = JekyllServer.new(context)
-          jekyll_server.execute
-        end
-        @thread.abort_on_exception = true
-
-        JekyllServe.mutex.synchronize do
-          running = JekyllServe.running?
-          JekyllServe.run_cond.wait(JekyllServe.mutex) unless running
-        end
+    data_dir = File.join(__dir__, '..', '..', '..', 'tests', 'minimal')
+    
+    before(:all) do
+      @thread = Thread.new do
+        context = Context.new('development', __dir__, data_dir)
+        jekyll_config_provider = JekyllConfigProvider.new(context)
+        context.configuration = jekyll_config_provider.provide('serve')
+        jekyll_server = JekyllServer.new(context)
+
+        jekyll_server.execute
       end
+      @thread.abort_on_exception = true
 
-      after(:each) do
-        JekyllServe.shutdown
+      JekyllServe.mutex.synchronize do
+        running = JekyllServe.running?
+        JekyllServe.run_cond.wait(JekyllServe.mutex) unless running
+      end
+    end
+
+    after(:all) do
+      JekyllServe.shutdown
+
+      JekyllServe.mutex.synchronize do
+        running = JekyllServe.running?
+        JekyllServe.run_cond.wait(JekyllServe.mutex) if running
+      end
+      FileUtils.remove_dir(File.join(data_dir, '_site') , force = true)
```

Comment:
Codacy found an issue: [Useless assignment to variable - `force`.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5820027)